### DISCUSSION TITLE
can we make it work with javascript off?

### DISCUSSION THREAD

**Discussion by abhinavpatel0** [PxlSyl/tailwind-nextjs-starter-blog-i18n#14] [General]
i was just interested to make it work when javascript is off , do anyone have any idea how can we do this?

**Comment by PxlSyl** [comment]
Hi, what do you mean by"javascript is off"? This repository is based on javascript frameworks (Node.js, React/Next)

**Comment by abhinavpatel0** [comment]
https://tailwind-nextjs-starter-blog.vercel.app/

open this there in browser settings of site  and off JavaScript, it will work because it actually supports Static Site Generation, where pages are pre-built at build time, but this template have nexti18n, which is preventing for this website to work without js

**Comment by abhinavpatel0** [comment]
i was wondering if there is any way to fix it

**Comment by PxlSyl** [comment]
Ok, I understand now. Can't help with this, maybe someone knows
